app/views/articles/show.html.erb

```diff
@@ -13,5 +13,11 @@
   <%= @article.status %>
 </p>
 
+<% if @article.status["draft"] %>
```

Comment:
```suggestion
<% if @article.status.draft? %>
```

Reply:
i edit enumeration with predicates option. It works rn